google_sheets_pkg/sheet_utils.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"golang.org/x/oauth2/google"
+	"golang.org/x/xerrors"
+	"google.golang.org/api/option"
+	"google.golang.org/api/sheets/v4"
+	"strings"
+)
+
+func ValidateInputAndGetConf(input InputBase, readOnly bool) (*sheets.Service, error) {
```

Comment:
I think this should just be named GetSheetsService

Reply:
updated